patches/minecraft/net/minecraft/world/entity/player/Player.java.patch

```diff
@@ -194,33 +194,14 @@
                       this.m_21008_(p_36159_, ItemStack.f_41583_);
                    }
  
-@@ -1020,7 +_,7 @@
-          double d1 = p_36201_.f_82481_;
-          double d2 = 0.05D;
- 
--         while(d0 != 0.0D && this.f_19853_.m_45756_(this, this.m_20191_().m_82386_(d0, (double)(-this.f_19793_), 0.0D))) {
-+         while(d0 != 0.0D && this.f_19853_.m_45756_(this, this.m_20191_().m_82386_(d0, (double)(-this.getStepHeight()), 0.0D))) {
-             if (d0 < 0.05D && d0 >= -0.05D) {
-                d0 = 0.0D;
-             } else if (d0 > 0.0D) {
-@@ -1030,7 +_,7 @@
-             }
-          }
- 
--         while(d1 != 0.0D && this.f_19853_.m_45756_(this, this.m_20191_().m_82386_(0.0D, (double)(-this.f_19793_), d1))) {
-+         while(d1 != 0.0D && this.f_19853_.m_45756_(this, this.m_20191_().m_82386_(0.0D, (double)(-this.getStepHeight()), d1))) {
-             if (d1 < 0.05D && d1 >= -0.05D) {
-                d1 = 0.0D;
-             } else if (d1 > 0.0D) {
-@@ -1040,7 +_,7 @@
-             }
-          }
+@@ -1014,6 +_,7 @@
+       return !this.f_36077_.f_35935_;
+    }
  
--         while(d0 != 0.0D && d1 != 0.0D && this.f_19853_.m_45756_(this, this.m_20191_().m_82386_(d0, (double)(-this.f_19793_), d1))) {
-+         while(d0 != 0.0D && d1 != 0.0D && this.f_19853_.m_45756_(this, this.m_20191_().m_82386_(d0, (double)(-this.getStepHeight()), d1))) {
-             if (d0 < 0.05D && d0 >= -0.05D) {
-                d0 = 0.0D;
-             } else if (d0 > 0.0D) {
++   // Don't update this method to use IForgeEntity#getStepHeight() - https://github.com/MinecraftForge/MinecraftForge/issues/8922
```

Comment:
This should start with `// FORGE: `, so it's easier to find all Forge-added comments by searching for `FORGE:`.

Reply:
fixed